physionet-django/user/views.py

```diff
@@ -766,7 +766,7 @@ def training_report(request, training_id):
     Serve a training report file
     """
     trainings = Training.objects.all()
-    if not request.user.is_admin:
+    if not request.user.has_perm('user.change_credentialapplication'):
```

Comment:
Not sure it really matters, but should this be `user.change_training`?

Reply:
Thank you.
Thats a good point. I remember we were using credentialing along with training, and want to separate it by adding new permission.

Do you think it would be a good idea to add the permission for training in this PR? as we would have to go ahead and make changes to templates and maybe few tests. Would it make more sense to open a separate PR for the training permissions?

I remember Ben suggested to not introduce multiple changes in a single PR and keep PR's short.

Please let me know what you recommend @tompollard , i can then (1)  push changes here or (2) open another PR after this or (3) open a PR side by side and squash some of the changes on training here.

Reply:
Oh sorry, i missed that. let me make changes to use it. Thank you :)

Reply:
Noted. resolving Comments in favour of opening another PR for trainings